interfaces/uss_qualifier/uss_qualifier.yaml

```diff
@@ -104,29 +91,35 @@ components:
                     type: object
                     description: >-
                         Content of test runs' json output file.
-            required:
-                - user_id
-                - test_id
-                - specification
-
-        TestsResponse:
-            type: array
-            items:
-                $ref: "#/components/schemas/TestRunResponse"
+        ListTestRunsResponse:
+            type: object
+            properties:
+                test_runs:
```

Comment:
Automatic code generation tools sometimes have problems with a message body that is an array rather than an object, so it's best to have root-level data structures be objects rather than arrays to maximize compatibility.